CI_YAML.md

```diff
@@ -76,6 +76,49 @@ promote a test that has been passing for the past 50 runs.
 To prevent tests from rotting, all targets are required to have a clear owner. Add an
 owner in [TESTOWNERS](https://github.com/flutter/flutter/blob/master/TESTOWNERS)
 
+## Properties
+
+Targets support specifying properties that can be passed throughout infrastructure. The
+following are a list of keys that are reserved for special use. Note, properties is a
+Map<String, String> and any special values can be JSON encoded.
+
+**add_recipes_cq**: String boolean whether to add this target to flutter/recipes CQ. This ensures
+changes to flutter/recipes pass on this target before landing.
+
+**caches**: JSON representation of swarming caches the bot running the target should have.
+Name is what flutter/recipes will refer to it as, and path is where it will be stored on the bot.
+Path should be versioned.
```

Comment:
What does this last bit mean? I don't see anything that looks like a version in the example, which seems very confusing.

Reply:
I added some more color, to the doc.

We've now had LUCI long enough that we are starting to need to support versioning dependencies. All the original deps and caches are not versioned, so we upgrade deps they will become versioned (see below an example for openjdk 11)

```
      caches: >-
        [
          {"name": "openjdk", "path": "java11"}
        ]
```